src/ResourceManager/Sql/Commands.Sql/Auditing/Model/AuditingPolicyModel.cs

```diff
@@ -46,11 +46,6 @@ public abstract class AuditingPolicyModel
         public string ServerName { get; set; }
 
         /// <summary>
-        /// Gets or sets the audit type
-        /// </summary>
-        public AuditType AuditType { get; set; }
```

Comment:
@ranisha2 @jaredmoo for the cmdlets that return an `AuditingPolicyModel` type, can we change the `OutputType` attribute to the subclass that is returned (be it `DatabaseAuditingPolicyModel` or `ServerAuditingPolicyModel`)? That will give users a better indication of the object they are returning, and there won't be confusion about whether or not the removal of this `AuditType` property is a breaking change.